src/control/cmd/daos/container.go

```diff
@@ -719,7 +700,7 @@ func newContainerInfo(poolUUID, contUUID *uuid.UUID) *containerInfo {
 	ci.LatestSnapshot = (*uint64)(&ci.dci.ci_lsnapshot)
 	ci.RedundancyFactor = (*uint32)(&ci.dci.ci_redun_fac)
 	ci.NumSnapshots = (*uint32)(&ci.dci.ci_nsnapshots)
-	ci.HighestAggregatedEpoch = (*uint64)(&ci.dci.ci_hae)
+	*ci.NumSnapshots = 0
```

Comment:
FWIW, I don't think this is necessary. When Go initializes a struct it always zeroes the fields: https://golang.org/ref/spec#The_zero_value ... Have you found that it's been printing garbage otherwise?

Reply:
OK thanks, good to know. I did not see any particular problem. I'll remove the line